.circleci/config.yml

```diff
@@ -21,24 +21,26 @@ jobs:
           command: ./build_and_push.sh
           environment:
             TAG_SUFFIX: ""
-            CAIRO_VERSION: 0.11.2
+            CAIRO_VERSION: "v0.11.2"
             OZ_VERSION: 0.5.1
             CAIRO_COMPILER_TARGET_TAG: "v1.1.0"
+            CAIRO_COMPILER_ASSET_NAME: "release-x86_64-unknown-linux-musl.tar.gz"
             SCARB_VERSION: 0.4.0
   build_push_arm:
-    machine:
-      image: ubuntu-2004:202111-02
-    resource_class: arm.medium
+    macos:
+            xcode: 14.2.0
+    resource_class: macos.m1.large.gen1
     steps:
       - checkout
       - run:
           name: Build and push image
           command: ./build_and_push.sh
           environment:
             TAG_SUFFIX: -arm
-            CAIRO_VERSION: 0.11.2
+            CAIRO_VERSION: "v0.11.2"
             OZ_VERSION: 0.5.1
             CAIRO_COMPILER_TARGET_TAG: "v1.1.0"
+            CAIRO_COMPILER_ASSET_NAME: "release-x86_64-unknown-linux-musl.tar.gz"
```

Comment:
Should we be using that asset for mac?

Reply:
I think as long as we're using an alpine image for the docker build, we can't proceed with the Mac binary (`release-aarch64-apple-darwin.tar`) but I have checked by compiling a cairo1 contracts and works fine.